cmd/build-helm.go

```diff
@@ -29,6 +30,7 @@ var buildHelmCmd = &cobra.Command{
 		flagBuildHelmTagExtra = buildHelmViper.GetString("tag-extra")
 		flagBuildHelmUseSecretsGenerator = buildHelmViper.GetBool("use-secrets-generator")
 		flagBuildOutputGraph = buildViper.GetString("output-graph")
+		flagBuildHelmAuthType = buildViper.GetString("auth-type")
```

Comment:
Shouldn't that be `buildHelmViper` instead of `buildViper` here?

Reply:
Fixed in 6d25908